build/props/common.props

```
@@ -17,9 +17,9 @@
             - Added nint overloads for the cases where OpenGL represents a native-sized integer as a void pointer.
             - Added support for executing Windowing loops during repaint and drag events on GLFW in some cases. (thanks @otac0n)
             - Added a distinction between "unset" (-1) and "default" (null) for Windowing depth/stencil buffer bits, defaulting to 24/8 on all platforms unless overridden.
-            - Added support for 64-bit Arm Linux for Assimp, GLFW (+ Windowing &amp; Input), OpenAL Soft, SDL, Vulkan Loader, SwiftShader, and WGPU. (thanks @alexrp)
+            - Added support for 64-bit Arm Linux for Assimp, GLFW (+ Windowing &amp; Input), OpenAL Soft, SDL, Vkd3d, Vulkan Loader, SwiftShader, and WGPU. (thanks @alexrp)
             - Added support for Apple Silicon Macs for OpenAL Soft, Vulkan Loader, and SwiftShader. (thanks @alexrp)
-            - Added support for 32-bit Arm Linux for Assimp, GLFW, OpenAL Soft, SDL, Shaderc, SPIR-V Cross, SPIR-V Reflect, SwiftShader, and WGPU. (thanks @alexrp)
+            - Added support for 32-bit Arm Linux for Assimp, GLFW, OpenAL Soft, SDL, Shaderc, SPIR-V Cross, SPIR-V Reflect, Vkd3d, SwiftShader, and WGPU. (thanks @alexrp)
             - Added support for 64-bit Windows on Arm for Assimp, OpenAL Soft, VulkanLoader, and SwiftShader. (thanks @alexrp)
             - Added native library support for glibc versions as old as 2.17 on Linux platforms.
```

The situation is a bit more nuanced:

* SPIR-V Cross, SPIR-V Reflect, and Shaderc are compiled for glibc 2.17 (built with Zig w/o linking to system libraries).
* SDL is compiled for glibc 2.27 (built on Ubuntu 18.04).
* DXVK is compiled for whatever glibc version Arch Linux is currently on (so 2.39 at the moment, AFAICT).
    * This is actually pretty bad now that I think about it...
* Everything else is compiled for glibc 2.35 (built on Ubuntu 22.04).

---

Thanks!
```suggestion
            - Added limited native library support (SPIR-V Tools and Shaderc) for glibc versions as old as 2.17 on Linux platforms.
```
We've never advertised glibc versions for the other libs so these will effectively be unchanged compared to the previous release.